docs/leaderboard/develop/api-reference/language_support.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_position: 2
+sidebar_label: Language Support
+title: Language API Support Matrix
+description: A reference as to which Momento SDK languages support each API
```

Comment:
Let's restate this more casual/reader friendly.
`See which Momento SDKs support the Leaderboard service`

Something like that.